<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SkinCenter</class>
 <widget class="QDialog" name="SkinCenter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>282</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="btn_close">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>7</y>
     <width>26</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="iconLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>25</y>
     <width>17</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>25</y>
     <width>231</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Local skin</string>
   </property>
  </widget>
  <widget class="QLabel" name="label1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>102</width>
     <height>68</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>60</y>
     <width>102</width>
     <height>68</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>60</y>
     <width>102</width>
     <height>68</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label4">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>60</y>
     <width>102</width>
     <height>68</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label8">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>150</y>
     <width>102</width>
     <height>68</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>102</width>
     <height>68</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label6">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>150</y>
     <width>102</width>
     <height>68</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label7">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>150</y>
     <width>102</width>
     <height>68</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="preBtn">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>240</y>
     <width>94</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="nextBtn">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>240</y>
     <width>94</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
